src/app/app.component.ts

```diff
@@ -8,20 +9,48 @@ import { takeUntil } from 'rxjs/operators';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent implements OnDestroy {
-  title = 'ng-resume';
+export class AppComponent implements OnInit, OnDestroy {
+  title = 'NgResume';
   activeId = '';
 
+  private readonly description =
```

Comment:
We could move this to a constant class, can do later

Reply:
Modified haha.. :)